recipes/feelnc/meta.yaml

```diff
@@ -21,6 +21,7 @@ requirements:
     - perl-bio-featureio
 
   run:
+    - perl ==5.22.0.1
```

Comment:
my dear @cmonjeau, can you left it as it was before? I think just rebuilding should be enough
BUT you'll need to remove feelnc from the blacklist https://github.com/bioconda/bioconda-recipes/blob/master/build-fail-blacklist#L1964